### DISCUSSION TITLE
Prompt management with UI

### DISCUSSION THREAD

**Discussion by nileshtrivedi** [BerriAI/litellm#4852] [Ideas]
It would be very useful to have a web-based UI for managing and iterating on prompt templates. 

I noticed [a PR was merged for LangFuse](https://github.com/BerriAI/litellm/issues/1994) but I don't understand how it is supposed to work.

**Comment by ghost** [comment]
Hey @nileshtrivedi what do you use today for prompt management? happy to support your existing provider

**Reply by ghost** [reply]
we support tracing the langfuse prompt, are you asking to inject their prompt template on incoming requests to the proxy?

**Reply by nileshtrivedi** [reply]
We're not using a proper tool yet. We created a `prompt_templates` table in our database and edit it via Supabase Database editor UI.

**Reply by nileshtrivedi** [reply]
<img width="581" alt="image" src="https://github.com/user-attachments/assets/714cddc7-9b1f-452a-9670-4ff13cc98b7c">
